src/components/Modal/Modal.tsx

```diff
@@ -114,6 +114,19 @@ export const Modal = forwardRef<HTMLDivElement, ModalProps>(function Modal(
   },
   ref
 ) {
+  if (
```

Comment:
this is great! i think it might also be good to only show this on dev builds, mui does something like: https://github.com/mui/material-ui/blob/512896973499adbbda057e7f3685d1b23cc02de9/packages/mui-styles/src/getStylesCreator/getStylesCreator.js#L8